gradle/libs.versions.toml

```diff
@@ -45,7 +45,7 @@ androidx-biometric = { module = "androidx.biometric:biometric", version.ref = "b
 androidx-core-ktx = { module = "androidx.core:core-ktx", version.ref = "core-ktx" }
 android-compose-bom = { module = "androidx.compose:compose-bom", version.ref = "bom" }
 android-kotlin-bom = { module = "org.jetbrains.kotlin:kotlin-bom", version.ref = "kotlinBom" }
-compose-ui = { module = "androidx.compose.ui:ui" }
+compose-ui = "androidx.compose.ui:ui:1.6.0-alpha08"
```

Comment:
Can you please write this in proper format?

Reply:
I need to use the Alpha version of the UI lib for the screenshot functionality, as it is not available in stable. So I can't use the bom

Reply:
Okay got it